service/matching/config.go

```diff
@@ -218,7 +220,8 @@ func NewConfig(
 
 		ListNexusIncomingServicesLongPollTimeout: dc.GetDurationProperty(dynamicconfig.MatchingListNexusIncomingServicesLongPollTimeout, 5*time.Minute-10*time.Second), // Use -10 seconds so that we send back empty response instead of timeout
 
-		FrontendAccessHistoryFraction: dc.GetFloat64Property(dynamicconfig.FrontendAccessHistoryFraction, 0.0),
+		FrontendAccessHistoryFraction:   dc.GetFloat64Property(dynamicconfig.FrontendAccessHistoryFraction, 0.0),
+		QueryWorkflowTaskTimeoutLogRate: dc.GetFloatPropertyFilteredByNamespace(dynamicconfig.MatchingQueryWorkflowTaskTimeoutLogRate, 0.0),
```

Comment:
Why not filtered by task queue also?

Reply:
Not completely sure. But from reading the description in the task, we actually don't know which workflowID and task queue for the customer were troublesome. So, I'm guessing we can't filter this config by task queue (?). The requirement is to log the workflow ID and the task queue so that we can troubleshoot easily.
@yycptt thoughts?

Reply:
ok. Sounds good. So what you're saying is it's harmless to filter by ns+task_queue since you can always skip specifying task queue in the configs?

Reply:
exactly. filtering by ns+task queue allows any of: ns+tq+tqtype, ns+tq, tq (shouldn't be used), ns, global.
see https://github.com/temporalio/temporal/blob/main/common/dynamicconfig/collection.go#L510-L521

Reply:
+1, also filtering on task queue make sense. OSS users can still see the task queue tag in the metrics I think.